doc/nrf/software_maturity.rst

```diff
@@ -216,6 +216,15 @@ The following table indicates the software maturity levels of the support for ea
 
   .. sml-table:: zigbee
 
+Wi-Fi feature support
+**********************
+
+The following table indicates the software maturity levels of the support for each Wi-Fi feature:
+
+.. toggle::
+
+  .. sml-table:: wifi
```

Comment:
I suspect this change will have to wait until the change to `software_maturity_features.yaml` have been merged, as this directive retrieves data from an online cache which must be updated with the values provided in that file.

Reply:
ok, so, two-phase process, will raise this in another PR.